### DISCUSSION TITLE
Add option to Attack Style plugin to prevent attacks with warned styles

### DISCUSSION THREAD

**Discussion by rcisnotfun** [runelite/runelite#18255] [Ideas]
Pickpocket Guardian plugin has the ability to prevent a player’s click to pickpocket going through. 

I think it would be useful to have a similar functionality available on the Attack Styles plugin that prevents a click to attack from going through if a warned style is somehow turned on. 

Perhaps this is redundant with the warned attack style red text, but this suggestion could automatically prevent an accident due to a misclick or general lack of awareness.

**Comment by cdfisher** [comment]
Just use "Remove warned styles" and then this isn't really necessary.

**Reply by rcisnotfun** [reply]
Perhaps. I usually have them removed but I know I’ve accidentally gotten xp before through some sort of user error resulting in an unexpected attack style, mostly when equipping a new weapon or just being careless on a new character. I think the game is supposed to remember attack style for either the specific weapon or weapon type, so maybe my issue can just be solved by equipping one of every weapon type and swapping to appropriate style. And using my eyeballs more effectively of course. 

I still think it’d be cool to have an automated safety net for those like me with a tendency for user error, but you are correct that removing warned styles can prevent a majority of issues.
